packages/@tinacms/toolkit/src/packages/fields/plugins/GroupListFieldPlugin.tsx

```diff
@@ -188,7 +211,9 @@ const Item = ({ tinaForm, field, index, item, label, ...p }: ItemProps) => {
               <GroupLabel>{title}</GroupLabel>
               <BiPencil className="h-5 w-auto fill-current text-gray-200 group-hover:text-inherit transition-colors duration-150 ease-out" />
             </ItemClickTarget>
-            <ItemDeleteButton onClick={removeItem} />
+            {(!fixedLength || (fixedLength && !isMin)) && (
```

Comment:
I kind of feel this might be a bit confusing to remove the button rather than just showing an error when we're below the minimum. (Otherwise, the user wouldn't have an error at this point, so there wouldn't be an indication of why they couldn't delete a list-item).

Alternatively, Maybe we could disable the button with a tooltip? (Same goes for the add-button)

Reply:
This only removes the delete button if the field has min and max set to the same amount and the field meets those requirements, since you'd never be able to delete an item. In other cases it's only ever disabled to indicate you can't currently delete an item but you'd otherwise be able to if you were above the minimum amount.